webui/src/pages/setup/index.jsx

```diff
@@ -26,30 +25,17 @@ const SetupContents = () => {
         }
     }, [error, response]);
 
-    const onSubmitAdminUser = useCallback(async () => {
-        setDisabled(true);
-        try {
-            const response = await setup.lakeFS(usernameRef.current.value);
-            setSetupError(null);
-            setSetupData(response);
-        } catch (error) {
-            setSetupError(error);
-            setSetupData(null);
-        } finally {
-            setDisabled(false);
-        }
-    }, [setDisabled, setSetupError, setSetupData, setup]);
-
-    const onSubmitCommunicationPreferences = useCallback(async (userEmail, updatesChecked, securityChecked) => {
+    const onSubmitCommunicationPreferences = useCallback(async (adminUser, userEmail, updatesChecked, securityChecked) => {
```

Comment:
I'd suggest renaming this function so it's clear that it submits all of the setup data

Reply:
done